Updated Zenodo Format Tests Instructions (#35)
Imageomics/Imageomics-guide: docs/wiki-guide/DOI-Generation.md

A `.zenodo.json` can be created by applying [cffconvert](https://github.com/citation-file-format/cffconvert) to your `CITATION.cff` (without the references, as these are not supported). Then add the references and other metadata back in to the JSON (following the [Zenodo dev guide](https://developers.zenodo.org/#representation)).
The `publication_date` and `version` will need to be updated along with the `CITATION.cff` for each release.
!!! example "Example `.zenodo.json`"

>     An example of this is the [Andromeda Zenodo JSON](https://github.com/Imageomics/Andromeda/blob/main/.zenodo.json), which was created in [this PR](https://github.com/Imageomics/Andromeda/pull/109). Include format tests following the example in this [PR](https://github.com/Imageomics/Andromeda/pull/120).

Reviewer: ```suggestion
    An example of this is the [Andromeda Zenodo JSON](https://github.com/Imageomics/Andromeda/blob/main/.zenodo.json). We also suggesting including format tests following the example in this [PR](https://github.com/Imageomics/Andromeda/pull/120).
```
